src/components/ViewAdmin.vue

```diff
@@ -21,16 +21,25 @@
 		</SettingsSection>
 		<SettingsSection
 			:title="t('recognize', 'Image tagging')">
+			<p>
+				<label>
+					<input v-model="settings['faces.enabled']" type="checkbox" @change="onChange">
+					<span>Enable face recognition</span>
+				</label>
+			</p>
 			<p>
 				<label>
 					<input v-model="settings['imagenet.enabled']" type="checkbox" @change="onChange">
 					<span>Enable object recognition</span>
 				</label>
 			</p>
-			<p>
+			<p style="margin-left: 20px;">
```

Comment:
Hmm, maybe we should use a class? :smile:

Reply:
yeah, lets do that. I have a tendency to avoid classes if it's only one property and I don't need to reuse it, but classes are still a bit cleaner :D

Reply:
I totally understand that, I have the same tendency but I know this is bad :laughing: